components/ui/select.tsx

```
@@ -1,65 +1,318 @@
+      value={{
+        selectedKeys: internalSelectedKeys,
+        onSelectionChange: handleSelectionChange,
+        isOpen,
+        setIsOpen,
+      }}
+    >
+      <div className={cn("relative", className)} ref={selectRef}>
+        {label && (
+          <label className="block text-sm font-medium text-gray-700 dark:text-gray-300 mb-1">
+            {label}
+          </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)